css/app.css

```
@@ -326,9 +328,27 @@ html {
     #main-content.active {
         margin-top: 230px;
     }
+    .banner-container {
+        height: 480px;
+    }
+    .banner-text {
+        font-size: 36px;
+        font-weight: 400;
+    }
```

we change the the font-size to 50px instead. then those media queries at a higher resolution will retain this style.
```suggestion
    .banner-text {
        font-size: 50px;
        font-weight: 400;
    }
```